src/bip44.c

```diff
@@ -103,7 +103,7 @@ uint32_t bip44_getChainTypeValue(const bip44_path_t* pathSpec)
 	return pathSpec->path[BIP44_I_CHAIN];
 }
 
-bool bip44_hasValidChainTypeForAddress(const bip44_path_t* pathSpec)
+static bool bip44_hasValidChainTypeForAddress(const bip44_path_t* pathSpec)
```

Comment:
why do we need static here?

Reply:
We don't, it's just a rather specific function used only here, so `static` is here to avoid leaking the name outside of this compilation unit. (Many of our .c files consistently use static for this purpose, but I think not all of them.)